go.mod

```diff
@@ -246,15 +248,24 @@ replace (
 	github.com/ghodss/yaml => github.com/ghodss/yaml v1.0.0
 	github.com/go-acme/lego => github.com/go-acme/lego v2.5.0+incompatible
 	github.com/go-bindata/go-bindata => github.com/go-bindata/go-bindata v3.1.1+incompatible
+	github.com/go-errors/errors => github.com/go-errors/errors v1.0.1
 	github.com/go-gl/glfw/v3.3/glfw => github.com/go-gl/glfw/v3.3/glfw v0.0.0-20200222043503-6f7a984d4dc4
 	github.com/go-kit/kit => github.com/go-kit/kit v0.9.0
 	github.com/go-logfmt/logfmt => github.com/go-logfmt/logfmt v0.4.0
 	github.com/go-logr/logr => github.com/go-logr/logr v0.4.0
+	github.com/go-openapi/analysis => github.com/go-openapi/analysis v0.19.5
+	github.com/go-openapi/errors => github.com/go-openapi/errors v0.19.2
 	github.com/go-openapi/jsonpointer => github.com/go-openapi/jsonpointer v0.19.3
 	github.com/go-openapi/jsonreference => github.com/go-openapi/jsonreference v0.19.3
-	github.com/go-openapi/spec => github.com/go-openapi/spec v0.19.3
+	github.com/go-openapi/loads => github.com/go-openapi/loads v0.19.4
+	github.com/go-openapi/runtime => github.com/go-openapi/runtime v0.19.4
+	github.com/go-openapi/spec => github.com/go-openapi/spec v0.19.5
+	github.com/go-openapi/strfmt => github.com/go-openapi/strfmt v0.19.5
 	github.com/go-openapi/swag => github.com/go-openapi/swag v0.19.5
+	github.com/go-openapi/validate => github.com/go-openapi/validate v0.19.8
```

Comment:
Please open a kustomize issue to track dropping the `go-openapi/{analysis,validate,strfmt}` deps in a future release... these have really questionable transitive dependencies (like a mongodb driver(!) - go.mongodb.org/mongo-driver) that are getting added here.

Those deps prompted extracting the bits needed for openapi validation of custom resources out to https://github.com/kubernetes/kube-openapi/tree/master/pkg/validation - if that has what you need to switch over (or a couple small additions would enable you to switch to it) that would be a much smaller dependency set.

Reply:
(and please link the issue back here once it's open)

Reply:
Do you recommend (alas) vendoring all deps in-repo again?  
Seems the simplest way to establish deny lists on the transitive deps.  

@dims Are you guys happy with the kubernetes tooling for this - and btw where is it? :-)

Reply:
Opened https://github.com/kubernetes-sigs/kustomize/issues/3705

Reply:
and https://github.com/kubernetes-sigs/kustomize/issues/3706

Reply:
@monopole we are getting someone to write proper tools for surfacing stuff like we caught here ( by visual inspection :) )